ruby/neo4j/driver/internal/async/inbound/channel_error_handler.rb

```diff
@@ -0,0 +1,80 @@
+module Neo4j::Driver
+  module Internal
+    module Async
+      module Inbound
+        class ChannelErrorHandler
```

Comment:
@klobuczek 
in java 

```
import io.netty.channel.ChannelInboundHandlerAdapter;
public class ChannelErrorHandler extends ChannelInboundHandlerAdapter
```

How should I handle this in ruby

Reply:
```suggestion
        class ChannelErrorHandler < Java::IoNettyChannel::ChannelInboundHandlerAdapter
```
see https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#referencing-java-classes-using-full-qualified-class-name

Reply:
Hello @klobuczek 
I have tried to handle java classes as you mentioned above but it throws the following error

```
Failure/Error: loader.eager_load

NameError:
  undefined local variable or method `io' for Neo4j::Driver::Internal::Async::Connection::BoltProtocolUtil:Class
```

Reply:
Hello @klobuczek 
I have tried to handle the Java class as you have mentioned above but it throws the following error

```
Failure/Error: loader.eager_load

NameError:
  missing class name io.netty.buffer.Unpooled
```

for the following case

`import static io.netty.buffer.Unpooled;`